src/Components/Publishing/EditorialFeature/__tests__/EditorialFeature.test.tsx

```diff
@@ -1,4 +1,5 @@
 import { mount } from "enzyme"
+import "jest-styled-components"
```

Comment:
Should this be added to [jest_setup](https://github.com/artsy/reaction/blob/master/src/setup_jest.ts)? // cc @ds300